Sources/ShapedArray/Operations/pack.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jaap Wijnen on 10/08/2022.
+//
+
+extension ShapedArray {
+    static func pack(_ values: [ShapedArray], axis: Int = 0) -> ShapedArray {
```

Comment:
Here's the original documentation we should include and adjust appropriately

https://github.com/s4tf/s4tf/blob/a31f79a7a8e92ff708b1034286a23510b23af5f3/Sources/x10/swift_bindings/apis/RawOpsManual.swift#L2158-L2185

Reply:
As per https://github.com/swift-numerics-workgroup/swift-shaped-array/pull/3#discussion_r945154932, the documentation should come from a TensorFlow `_Raw` operation. The Swift API uses its own definition, which is worded differently:

https://github.com/s4tf/s4tf/blob/a31f79a7a8e92ff708b1034286a23510b23af5f3/Sources/TensorFlow/Initializers.swift#L124-L156